### ISSUE TITLE
Incorrect classes for empty rows, missing space

### ISSUE THREAD

**Issue by clse** [verbb/formie#932]
Formie version: 1.5.19

The classes for empty rows are incorrectly rendered, missing a space.

Appears as `fui-row fui-page-rowfui-row-empty` instead of `fui-row fui-page-row fui-row-empty`.

This causes empty rows to show padding on the front end since the class is not correctly applied.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.0](https://github.com/verbb/formie/releases/tag/1.6.0)
